pkg/router/f5/f5.go

```diff
@@ -525,11 +556,13 @@ func (f5 *f5LTM) ensureVserverHasPolicy(vserverName, policyName string) error {
 	glog.V(4).Infof("Checking whether vserver %s has policy %s...",
 		vserverName, policyName)
 
+	vserverUriPath := f5.iControlVserverUriPath(vserverName)
+
 	// We could use fmt.Sprintf("https://%s/mgmt/tm/ltm/virtual/%s/policies/%s",
 	// f5.host, vserverName, policyName) here, except that F5 iControl REST
 	// returns a 200 even if the policy does not exist.
 	vserverPoliciesUrl := fmt.Sprintf("https://%s/mgmt/tm/ltm/virtual/%s/policies",
-		f5.host, vserverName)
+		f5.host, vserverUriPath)
```

Comment:
Yeah, I just followed what we had using the f5host name  + the paths everywhere  (`/mgmt ... `),  we should probably just use something like : 
`getF5ResourceURI(resourceType, resourceName)`  -
then all the interpolation can be done in a single place.